Library/ViewModels/DiscoveryFiltersViewModel.swift

```diff
@@ -205,18 +205,24 @@ private func expandableRows(selectedRow selectedRow: SelectableRow,
           }
       )
     }
-    .sort { lhs, rhs in lhs.params.category < rhs.params.category }
+    .sorted { lhs, rhs in
+      guard let lhsCategory = lhs.params.category, let rhsCategory = rhs.params.category else {
+        return lhs.params.category == nil
+      }
+      return lhsCategory < rhsCategory
```

Comment:
this is me unrolling the optional comparison stuff. luckily had tests verifying all this jazz! i got it mixed up the first few times...